testing/cypress/e2e/integration-900-deleteIntegration.cy.ts

```diff
@@ -10,32 +10,34 @@ const testData = [...data1, ...data2]; // Define the testData array
 console.log(testData);
 console.log('Right One');
 
-describe('Delete Integration Requests', () => {
-  beforeEach(() => {
-    // Use admin to delete the requests as there might be a mix of requests created by regular user and admin
-    cy.setid('admin').then(() => {
-      cy.login(null, null, null, null);
+if (!Cypress.env('localtest')) {
```

Comment:
Why do we need to skip this on the local run? Is it because of those manual creations from the readme?

Reply:
This should not affect the manual creations, the 900 delete reads the temporary file that the create integration populated with the IDs. The 990 delete only deletes those items that have @20240226 Pattern in their name

Reply:
@jlangy yeah the main reason I commented it out was because it always failed locally and the deletion is covered by 990.  This just saves a bit of time on the local run.